board/fischertechnik/TXT/rootfs/opt/ftc/apps/system/display/display.py

```diff
@@ -82,6 +86,21 @@ def restart_launcher(self, text):
     def do_restart_launcher(self):
         subprocess.run(["sudo", "/etc/init.d/S90launcher", "restart"])
 
+    def unset_reset_calibration_flag(self):
+        subprocess.run(["/sbin/unset-touchscreen-calibration-reset-flag"])
```

Comment:
See above. Simply delete the file directly from Python.

And drop the set/unset scripts as well.